packages/lesswrong/server/repos/UsersRepo.ts

```diff
@@ -688,6 +688,20 @@ class UsersRepo extends AbstractRepo<"Users"> {
       LIMIT $2
     `, [userId, limit]);
   }
+
+  async getSitemapUsers(): Promise<Pick<DbUser, "slug">[]> {
+    return this.getRawDb().any(`
+      -- UsersRepo.getSitemapUsers
+      SELECT "slug"
+      FROM "Users"
+      WHERE
+        NOT "noindex"
+        AND NOT "deleted"
+        AND "banned" IS NULL
+        AND "reviewedAt" IS NOT NULL
```

Comment:
There are a bunch of valid users where "reviewedAt" is null, presumably from a time before we required new users to be reviewed. These are the top karma ones:
```
slug | karma
linch | 2761
michael-st-jules | 2468
michaela | 2394
nathan | 2163
aaron-gertler | 1848
habryka-deactivated | 1385
larks | 1367
peter_wildeford | 1352
nicklaing | 1300
```
Could you change it to instead use the same condition for a user to show up in search (which I think is just this query without "reviewedAt", but worth checking)?